ls1intum/Artemis
# Enhancements for Exam Student Import

### Checklist

### Description
- Registration number was not shown right away when adding a student to the exam via the search bar
- More instructive error message when the csv can not be interpreted correctly

### Steps for Testing
1. Log in to Artemis
2. Navigate to Course Management
3. Click on Exams for a Course (This button is hidden, go to `course-management/{courseId}/exams`)
4. Create an Exam if needed
5. Click on Students for an Exam
6. Search for a student to add them to the Exam
7. Check that matriculation number is shown in table column
8. Click on "Import students" and select a csv file that contains an error (sample file with typo in header column: [students.txt](https://github.com/ls1intum/Artemis/files/4820582/students.txt), needs to be renamed to .csv)
9. Check that error message is shown as in screenshot below

### Screenshots
![OnPaste 20200623-180342](https://user-images.githubusercontent.com/7109225/85427991-d148c200-b57c-11ea-9d13-bb556d36c841.png)
